src/marketplace/sellers/marketing-review-guidelines.md

```diff
@@ -9,32 +9,83 @@ From a marketing perspective, all submissions to Magento Marketplace must solve
+   *  No slogans or features.
+*  Do not use _"Magento"_ or _"M2"_.
+*  Do not use Magento Logo.
+*  Do not use the developer logo here instead of the extension icon.
+*  Do not specify the extension version.
+*  Limited use of _"Pro"_.
+*  Less is more. Tiny, intricate details are not easy to make out, try to avoid them.
+*  For the 1st party companies, we recommend using the same image, as you are using on Twitter, LinkedIn, etc.
+
+## Long description
+
+For guides and examples of long descriptions, see [Product Descriptions]({{ site.baseurl }}/marketplace/sellers/product-descriptions.html).
+
+## User guide, reference manual or installation guide
+
+Extension documentation should fulfill the following guidelines:
+
+*  Extension documentation must cover all features of the extension.
+*  Extension documentation must not direct users to make purchases on sites other than Marketplace.
+*  If an extension title added into documentation, it must not include _"Magento"_ in it. Use _"For Magento"_ after the title, if needed.
+*  Use of the Magento logo is prohibited. To learn more, see our [branding guidelines]({{ site.baseurl }}/marketplace/sellers/branding.html).
+   *  Partners can use their Magento partner badge.
+*  At least one piece of uploaded documentation must be in English
+*  Do not upload duplicates of the documents.
 
 ## Pricing: Provide clear pricing information
 
 The price of your extension is included as part of the marketing information. To learn more, see [Enter Pricing Information]({{ site.baseurl }}/marketplace/sellers/submit-for-marketing-review.html).
 
--  Items listed on Magento Marketplace can be offered for free, or sold as a commercial products.
--  The price of items offered for sale on Magento Marketplace can range anywhere from $25.00 to $999,999.00, and must reflect fair market value.
--  All purchases are paid for in US currency.
--  Items sold on Magento Marketplace are subject to a revenue sharing fee that is paid by the developer.
--  Any additional fees or subscription pricing — including fees charged by a third-party company for integrated services — must be included in the opening paragraph of the Long Description with bold formatting.
--  Magento recommends the following extension pricing:<br />
-  Open Source - $99<br />
-  Commerce - $199
+*  Items listed on Magento Marketplace can be offered for free, or sold as a commercial products.
+*  The price of items offered for sale on Magento Marketplace can range anywhere from $25.00 to $999,999.00, and must reflect fair market value.
+*  All purchases are paid for in US currency.
+*  Items sold on Magento Marketplace are subject to a revenue sharing fee that is paid by the developer.
+*  Any additional fees or subscription pricing — including fees charged by a third-party company for integrated services — must be included in the opening paragraph of the Long Description with bold formatting.
```

Comment:
```suggestion
*  Any additional fees or subscription pricing—including fees charged by a third-party company for integrated services—must be included in the opening paragraph of the Long Description with bold formatting.
```

Reply:
You use "third-party" here but "1st party" higher up in the doc. Pick one convention and use the same for both instances.